src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -139,6 +139,25 @@ def test_linux_webapp_quick_create(self, resource_group):
             JMESPathCheck('[0].value', 'false'),
         ]))
 
+    @ResourceGroupPreparer()
+    def test_linux_webapp_multicontainer_create(self):
```

Comment:
Can you log in issue and assign to yourself, so you can improve the test to create all from the scratch instead of hard-coding the assert?